docs/NETWORK-MODELS.md

```diff
@@ -76,12 +76,31 @@ There is no connection directly to the off-device network; remote access is avai
 If an EVE device is instructed to connect ECOs via L3, it should do the following:
 
 1. Set up an L3 network on the device using a Linux bridge
+   * Checks if the existing Ethernet is not a bridge, If so rename it to **kethN** and create a bridge and name it **ethN** and move the MAC address. Reverse this actions if it's removed from DevicePortConfig(DPC)
+   * Bridges creating after DevicePortConfig changes and also after (re)boot when it receives initial config from the controller.
+   * At least one Ethernet port is assigned to EVE for management traffic
```

Comment:
While most of it is technically correct, I'm wondering how much of details we should go into (IOW how much really is just an implementation detail and shouldn't relied upon). @nordmark WDYT?

Reply:
This document isn't about how EVE configures the device network, but only about how EVE configures networking for the applications aka edge container workloads. Hence the creation/use of kethN is and DevicePortConfig is out of scope. A description for that would belong e.g., in some new implementation section in DEVICE-CONNECTIVITY.md

What makes sense here is strating that 1) for local network instances and cloud network instances, EVE creates a Linux bridge (named bnX), 2) for a switch network instance with e.g., ethN as the external port it will use the ethN Linux bridge and for a switch network instance without an external port it will create a Linux bridge (named bnX).

But note that all this doesn't belong in a section on local networks since it would describe all types.